src/style.css

```diff
@@ -2,0 +1,94 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
```

Comment:
Avoid using padding on the `body` tag to maintain consistency. If padding is needed, it's better to apply it to individual elements or containers.